### PR TITLE
Change Vietnamese subdivisions from an array to a hash

### PR DESCRIPTION
**Pull request by johnnymo87**

### CODE REVIEW

File: lib/data/subdivisions/VN.yaml [hunk 1]
```diff
@@ -1,194 +1,194 @@
...
+'28':
+  name: Kon Tum
+  names: Kon Tum
+'01':
+  name: Lai Chau
+  names: Lai Chau
+'35':
+  name: Lam Dong
+  names: Lam Dong
+09
```
Comment (bitsofdave) [inline lib/data/subdivisions/VN.yaml]: Should this key also be a string and have a `:` at the end?

File: lib/data/subdivisions/VN.yaml [hunk 2]
```diff
@@ -1,194 +1,194 @@
...
+'37':
+  name: Tay Ninh
+  names: Tay Ninh
+'20':
+  name: Thai Binh
+  names: Thai Binh
+'69':
+  name: Thai Nguyen
+  names:
     - Central Highlands
```
Comment (bitsofdave) [inline lib/data/subdivisions/VN.yaml]: Just wondering -- does the `-` in front of "Central Highlands" do anything?

File: lib/data/subdivisions/VN.yaml (hunk 1 above)
Comment (johnnymo87) [inline lib/data/subdivisions/VN.yaml]: Good catch, fixed.

File: lib/data/subdivisions/VN.yaml (hunk 2 above)
Comment (johnnymo87) [inline lib/data/subdivisions/VN.yaml]: I think this is list syntax. I'm not sure why this needs to be a list if it has only one element, but I see other subdivision files that do the same.

**Pull request merged**